src/libraries/Microsoft.Extensions.Logging.Abstractions/gen/LoggerMessageGenerator.Parser.cs

```diff
@@ -188,17 +188,16 @@ public IReadOnlyList<LoggerClass> GetLogClasses(IEnumerable<ClassDeclarationSynt
                                         break;
                                     }
 
-                                    IMethodSymbol? methodSymbol = sm.GetDeclaredSymbol(method, _cancellationToken);
-                                    if (methodSymbol != null)
+                                    if (logMethodSymbol != null)
```

Comment:
>if (logMethodSymbol != null) [](http://example.com/codeflow?start=36&length=28)

We already have this check as assert in the top. do we need to keep this check? or we can move this check replacing the assert in the top?

Reply:
Good catch, I will just remove it and keep just the Assert (since if this symbol was not found, there is a bug in our side)

Reply:
Thanks. Just ensure there is no nullability failures :-)